src/main/java/modtrekt/model/module/ModCode.java

```diff
@@ -9,8 +9,9 @@
  */
 public class ModCode implements Comparable<ModCode> {
 
-    public static final String MESSAGE_CONSTRAINTS = "Code should contain alphanumeric characters and no white space";
-    private static final String ALPHANUMERIC_NO_UNDERSCORE = "[^\\W_]+"; // alphanumeric characters except underscore
+    public static final String MESSAGE_CONSTRAINTS =
+            "Code should contain alphanumeric characters, no white space, and 6 to 9 characters long";
```

Comment:
Can we make this more specific, e.g.
`Code should only contain alphanumeric characters, should not contain white space and should be between 6 and 9 characters long`?

Reply:
done. the UG changes ill do it in the other PR #210